infection.json.dist

```diff
@@ -12,5 +19,6 @@
         }
     },
     "minMsi": 100.00,
-    "minCoveredMsi": 100.00
+    "minCoveredMsi": 100.00,
+    "testFrameworkOptions": "--testsuite=unit"
```

Comment:
Isn't it better to include all phpunit tests in the msi?

Reply:
My understanding of mutation tests is it's great for checking the edge cases not covered by unit tests - the functional tests would reflect the business value of the application and does not have to cover all the edge cases, as they could be covered on the unit level. And are just much more demanding, pretty heavy to mutate them. Again, as always, I'm open for the discussion 🖖

Reply:
Infection mutates the code, not the tests, and I really believe we shouldn't limit Infection just to increase the MSI. I would much rather have a low MSI that tries to mutate the whole code base, so that we can see where we have the 'worst' edge cases

Reply:
[OK](https://github.com/Setono/sylius-order-edit-plugin/pull/3#discussion_r1639809054) :)